packages/issuer-callback/lib/IssuerCallback.ts

```diff
@@ -0,0 +1,27 @@
+import * as didKeyDriver from '@digitalcredentials/did-method-key'
+import { Ed25519Signature2020 } from '@digitalcredentials/ed25519-signature-2020'
+import { Ed25519VerificationKey2020 } from '@digitalcredentials/ed25519-verification-key-2020'
+import { securityLoader } from '@digitalcredentials/security-document-loader'
+import vc from '@digitalcredentials/vc'
+import { CredentialRequest } from '@sphereon/openid4vci-common'
+import { ICredential, W3CVerifiableCredential } from '@sphereon/ssi-types'
+
+export const generateDid = async () => {
```

Comment:
Please add some inline comments that this is just an example and explain what it does

Reply:
Fixed!